### ISSUE TITLE
General issues (live dev, will be updated)

### ISSUE THREAD

**Issue by StellarisStudio** [humhub/humhub#2018]
Hi folks... As I just started to develop a "community social network" I will probably use this "general issue" for many small issues I face !! So It will be edited and/or updated depending my needs!!

**Questions :**
Regarding to the upcoming update (v1.2), I would like to know if themes and modules will merge properly ?? Or the core will be "a lot" different ? (cause in that case, what I do now is a waste of time)
And what about the "following stuff" ?? How it works? Do people can see private post of somone they follow but are not friend to... ??

**Some Issues :**
- I see that there is a way to have the total of comments for a post, but I didn't get the way to show it like the "like count"... Every test make the "stream" not loading (always the 3 points loader)! _(The idea is to show the number of comments as like does and change the way comments show, hide first always and then show comments on click on the icon. **That would be awesome ^^**)_
- How "page" get injected inside the theme... For example I create a page inside a folder inside the theme folder... And when trying to reach that page, I always fall on a kind of 404 error !!
- How to make stuff more "live", more dynamic ?! Like the notifications, the mails notifications, the like count... and more? They update only after page refreshing... Even the friends request on profile reload the profile page... (crazy ^^  why not just dynamically change the button.)
  By the way, it would be nice to split the "friend request notification" from the other one.. (fb style)!
- ReportContent Module doesn't work (https://github.com/humhub/humhub-modules-reportcontent/issues/19).
- As it is not in the original theme, I wonder if I can use "use yii\helpers\Url;" on the main.php (on my theme) ? (security?)....... And what is the difference between "Url::to" and "Url::toRoute" ??

**Some ideas :**
- That would be nice to have some setting for core module such as activate or not the topmenu in the sidebar (rather than touching the code itself), same for "recent activity" on dashboard....
- That would be nice to have a kind of index (repertory) of the "system" Url::toRoute (like: <?php echo Url::to(['/directory/directory']); ?>), and why not with their translation Yii::t (like: <?php echo Yii::t('DirectoryModule.base', 'Directory'); ?>) (I start one but still don't have it all).
- That will be nice to have a themes css with "comments"!!
- **Brief...** I need more documentation to work better... Right now I lose a lot of time in research (online and inside folders & files).......
  Juste for you to remember that _HumHub is 25000+ files inside 4 300+ folders_ .... Hard to start development without info!!

**_Thanks a lot for any info and help!!_**

> My wisdom flows from the highest Source... I salute that Source in you... Let us to work together!! (meaning of Namasté^^)"

**Comment by Lokyaziis** [comment]
Some answer please !!!!!!!!!

**Comment by Lokyaziis** [comment]
I don't get why, but the "activity panel" on dashboard breaks, for my member test.... As admin it works fine. ??? Strange...
